js/speakers.js

```diff
@@ -0,0 +1,86 @@
+const speakersArray = [
+  {
+    name: 'Mohamed Saleh',
+    title: 'CEO of Creative Engineers and Company Machine Learning Specialist',
+    description: "Mohamed is originally a communication engineer he gained his electrical engineering BA degree in 2016 and a master's degree in computer science from Arizona State University in 2020. Mohamed is a very technical guy who has expertise in various fields: Machine Learning, web development, embedded systems, and RF.",
+    image: 'images/me.jpeg',
+  },
+  {
+    name: 'Mostafa Hemada',
+    title: 'CTO of Creative Engineers and Company VLSI Specialist',
+    description: 'Mostafa is one of the most talented engineers in electronics Digital design he graduated from Zewail City university in 2018 with a BA in nanoelectronics engineering.Mostafa has a very good experience in using Matlab,  Simulink, and synopsis tools',
+    image: 'images/mybro.jpg',
+  },
+  {
+    name: 'Ahmed Kamal',
+    title: 'Cryptography company specialist',
+    description: 'Ahmed Kamal graduated from menoufia university with BA in communication and electronics engineering in 2013 he also got a master degree in communication engineering in 2022. ahmed has a very good expertise in dealing with ciscio devices and has a very good foundation in networking in general beside he has a very good expertise in cryptography and a very active researcher in that field his recent work was inventing new compression encryption scheme technique to be used in IOT devices.',
+    image: 'images/kemo.jpg',
+  },
+  {
+    name: 'Mostafa Elhadad',
+    title: 'Electrical Power engineering specialist',
+    description: 'Mostafa Elhadad graduted from Alexandria university with BA in Electrical Engineering in 2017. Mostafa has a very good Applicaple expertise with dealing with Siemiens devices and programs he also has a very good experience with using PLC, Scada and classical control. he also can literaly mentor building any power system in any scale start from home to big factories!',
```

Comment:
in the desktop version of you website, the speaker section must have two column each with six individual articles

your design 
![image](https://user-images.githubusercontent.com/60612329/181996900-67faf5da-4b5d-4970-8289-99f6cef23bd7.png)

vs 

the required design 

![image](https://user-images.githubusercontent.com/60612329/181996921-e9d7d823-8d72-4937-92ab-15624213270a.png)

Reply:
@alexander16108  sorry I didnot get it I have also 6 individual articles and s column within a row can you clarify?